common/jenkins-agents/python/docker/Dockerfile

```diff
@@ -5,45 +5,42 @@ LABEL maintainer="Gerard Castillo <gerard.castillo@boehringer-ingelheim.com>"
 ARG nexusHost
 ARG nexusAuth
 
-ENV PYTHON_VERSION=3.6.10
-ENV VENV_VERSION=20.0.9
-ENV INSTALL_PKGS="yum-utils gcc make openssl-devel zlib-devel sqlite-devel"
-
-RUN set -x \
-    && yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS \
-    && yum clean all \
-    && rm -rf /var/cache/yum/*
-
-RUN cd /tmp \
-    && curl -O https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz \
-    && tar xzf Python-${PYTHON_VERSION}.tgz -C / \
-    && rm -rf Python-${Python_VERSION}.tgz
-
-RUN cd /Python-${PYTHON_VERSION} \
-    && ./configure \
-    && make altinstall \
-    && ln -s /Python-${PYTHON_VERSION}/python /usr/local/sbin/python3 \
-    && python3 -V \
-    && chmod a+rx /Python-${PYTHON_VERSION} \
-    && chmod a+rx /Python-${PYTHON_VERSION}/python \
-    && yum remove -y $INSTALL_PKGS
```

Comment:
The `yum remove` is now missing. Will this make the image larger.

Reply:
right, but that is by purpose because we want to address #407 and let users be able to build packages within jenkins job if they need/want to